src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -1,36 +1,20 @@
 import * as Sentry from '@sentry/browser';
-
-import { ssoKeepAliveEndpoint } from 'platform/user/authentication/utilities';
+import {
+  SSO_KEEP_ALIVE_ENDPOINT,
+  CSP_AUTHN,
+  AUTHN_HEADERS,
+  CAUGHT_EXCEPTIONS,
+} from './constants';
 
 const SENTRY_LOG_THRESHOLD = [Sentry.Severity.Info];
 
 const logToSentry = data => {
   let isCaptured = null;
   const { message } = data;
-  const caughtExceptions = {
-    'Failed to fetch': {
-      LEVEL: Sentry.Severity.Warning,
-    },
-    'NetworkError when attempting to fetch resource.': {
-      LEVEL: Sentry.Severity.Warning,
-    },
-    'The Internet connection appears to be offline.': {
-      LEVEL: Sentry.Severity.Info,
-    },
-    'The network connection was lost.': {
-      LEVEL: Sentry.Severity.Info,
-    },
-    cancelled: {
-      LEVEL: Sentry.Severity.Info,
-    },
-    default: {
-      LEVEL: Sentry.Severity.Error,
-    },
-  };
 
-  const LEVEL = caughtExceptions[message]
-    ? caughtExceptions[message].TYPE
-    : caughtExceptions.default.TYPE;
```

Comment:
Looks like `.TYPE` never existed and this may have not been working for some time